Sources/Momento/internal/TopicsGrpcManager.swift

```diff
@@ -0,0 +1,50 @@
+import GRPC
+import NIO
+import NIOHPACK
+
+@available(macOS 10.15, iOS 13, *)
+internal class TopicsGrpcManager {
+    var channel: GRPCChannel
+    var client: CacheClient_Pubsub_PubsubAsyncClient
+    
+    init(
+        credentialProvider: CredentialProviderProtocol,
+        eventLoopGroup: EventLoopGroup,
```

Comment:
docs indicate a single event loop group can be shared by multiple Channels: https://swiftinit.org/docs/swift-nio/niocore/eventloop